bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -47,6 +49,15 @@
 				<description>Authentication Token which can be found under "Software Integration" if you connect locally to your
 					sonnen battery. If empty the old deprecated API will be used.</description>
 			</parameter>
+			<parameter name="chargingPower" type="integer" unit="W" min="0" max="10000">
+				<label>sonnen battery maximum charging power</label>
```

Comment:
Labels should be < 23 chars and 2-3 words proper capitalized.
```suggestion
				<label>Maximum Charging Power</label>
```

Reply:
ok